docs/pages/database-access/reference/configuration.mdx

```diff
@@ -23,15 +23,26 @@ The following Proxy service configuration is relevant for database access:
   type="warning"
   title="Proxy TLS Warning for PostgreSQL"
 >
-  The PostgreSQL connection requires TLS enabled for the SSL connection that operates on the `web_listen_addr`.
-  Do not set `--insecure-no-tls` for the proxy Teleport instances as a parameter.  If you are terminating TLS at a
-  Application Load Balancer (ALB) or other service that may require enabling a backend protocol of HTTPS for the target address.  
+The following database types require TLS for SSL connections, using the
+`web_listen_addr` value:
+
+- Postgres
+- MongoDB
+- Redis
+- CockroachDB
+- Elasticsearch
+- OpenSearch
```

Comment:
The Admonition title is "Proxy TLS Warning for PostgreSQL"  but this section quite generic to any  and refers to all DB protocols. 


Additional this list is not complete and quickly will be outdate because right now when we adding new db protocol the TLS listener must be enabled on Teleport Proxy side and as @greedy52 noted  the `--insecure-no-tls`  breaks almost all DB integrations apart from: 
- MySQL/Maria
- Postgres on separate port. 

Can we just say that `--insecure-no-tls` flag  in scope of Teleport Database Access is supported only with MySQL and Postgres (on separate listener `proxy_service.postgres_listen_addr` must provided ) integratons and other DB protocols requires to use TLS Routing routing where Teleport proxy TLS Listener needs to be enabled.